examples/simple-lambda-worker/lib/simple-lambda-worker-stack.ts

```diff
@@ -48,16 +59,24 @@ export class SimpleLambdaWorkerStack extends cdk.Stack {
         lambdaProps: {
           environment: {
             EXAMPLE_ENV_VAR: "example value",
+            SUCCESS_QUEUE_URL: successQueue.queueUrl,
           },
           entry: "src/lambda/simple-worker.js",
           handler: "simpleLambdaWorker",
           memorySize: 1024,
-          timeout: cdk.Duration.minutes(5),
+          timeout: cdk.Duration.seconds(30),
```

Comment:
30 seconds is the minimum. Reducing it because this affects how long it takes for the message to appear on the DLQ in the integration test.